packages/design-system/src/themes/default/index.css

```diff
@@ -103,4 +103,44 @@
   /** Uneditable Text Field */
   --ads-uneditable-text-field-label-text-color: var(--ads-color-black-750);
 
+
+
+  /* Variables V2 */
+
+  /** Foreground/text **/
+  --ads-color-fg-emphasis-plus: var(--ads-color-gray-800);
```

Comment:
re: fg/bg - please let us use full forms of things as much as we can. It improves readability and in the era of modern IDEs makes no discernible difference in typing time.